quirks/websites-with-shared-credential-backends.json

```diff
@@ -197,5 +197,14 @@
     [
         "lookmark.io",
         "lookmark.link"
+    ],
+    [
+        "claro.com.br",
+        "auth.netcombo.com.br",
+        "net.com.br"
+    ],
```

Comment:
There is a login page at each of these websites that lets you log in with the same credentials?

Reply:
Yes @rmondello Same credentials.

Reply:
Just for documentation purposes in case we ever have to re-visit this, can you provide URLs for the login pages at each of these domains? (I’m really excited about this PR and it’s very close.)

Reply:
For sure @rmondello :)

Here they are:

Everyone directs to auth.netcombo.com.br


https://minhaclaro.claro.com.br -> https://auth.netcombo.com.br/web/login.html?client_id=MinhaClaroDIG&response_type=code&redirect_uri=https%3A%2F%2Fminhaclaro.claro.com.br&scope=minha_claro_dig&authMs=UP,EP

https://minhanet.net.com.br -> https://auth.netcombo.com.br/web/login.html?client_id=MINHA_NET_WCP&redirect_uri=https%3A%2F%2Fminhanet.net.com.br%2Fwebcenter%2Fportal%2FNETAutoAtendimento&response_type=code&scope=openid+minha_net&state=db2433bb-fddb-4251-8321-3c12619b0c75&authMs=UP,EP

https://mc.claro.com.br -> https://auth.netcombo.com.br/web/login.html?client_id=MinhaClaroDIG&response_type=code&redirect_uri=https%253A%252F%252Fminhaclaro.claro.com.br&scope=minha_claro_dig&authMs=UP,EP

Reply:
So you can never type or fill the credentials into all of these websites? If these websites are all redirecting to one central domain for authentication, then they’re following a best practice, and we don’t need these rules. Users’ credentials will always be saved for auth.netcombo.com.br.